lib/cli_common/cli_common/phabricator.py

```diff
@@ -163,6 +163,28 @@ def comment(self, revision_id, message):
             attach_inlines=1,
         )
 
+    def load_parents(self, revision_phid):
```

Comment:
There's an implementation that might be faster (smaller amount of queries) that smacleod gave me a while ago: https://github.com/mozilla/code-coverage-addon/issues/6#issuecomment-382797648.
If you want to do it in a follow-up, can you file an issue?

Reply:
This code sample is only to get the first parents and their details, but will not reach older parents than depth=1.